packages/core/src/components/essentials/tooltip-scatter.ts

```diff
@@ -7,6 +7,10 @@ export class TooltipScatter extends Tooltip {
 		if (type === TooltipTypes.TITLE) {
 			// the main tooltip component handles title styles
 			return super.getTooltipHTML(datum, type);
+		} else if (type === TooltipTypes.LEGEND) {
+			return `<div class="legend-tooltip"><p class="label">${datum.name}</p></div>`;
+		} else if (type === TooltipTypes.AXISLABEL) {
+			return `<div class="axis-tooltip"><p class="label">${datum}</p></div>`;
```

Comment:
Tooltip should at some point be refactored to just handle showing a single value rather than being called many different types of tooltips, but this is a working solution for now so 💯